### PR TITLE
x11-wm/marco: add 1.26.1

### PR DESCRIPTION
**Pull request by oz123**
Signed-off-by: Oz Tiram <oz.tiram@gmail.com>

### CODE REVIEW

File: x11-wm/marco/marco-1.26.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (scardracs) [inline x11-wm/marco/marco-1.26.1.ebuild:4]: EAPI=8

File: x11-wm/marco/marco-1.26.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+MATE2_LA_PUNT="yes"
+
+inherit mate
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+fi
```
Comment (scardracs) [inline x11-wm/marco/marco-1.26.1.ebuild:12]: `KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"` (drop the if...fi as live ebuild does not exist anymore)

**Review by scardracs — CHANGES_REQUESTED** [review]

File: x11-wm/marco/marco-1.26.1.ebuild (hunk 2 above)
Comment (oz123) [reply x11-wm/marco/marco-1.26.1.ebuild:12]: done.

File: x11-wm/marco/marco-1.26.1.ebuild (hunk 1 above)
Comment (oz123) [reply x11-wm/marco/marco-1.26.1.ebuild:4]: done..

**Pull request closed without merging**